### ISSUE TITLE
Adding author's name(s) on idea page and card

### ISSUE THREAD

**Issue by PierreMesure** [mainio/decidim-module-ideas#11]
Right now, the author's name isn't present anywhere on the idea card nor the idea page. That seems to be a choice, not a mistake.

Do you have any plans of adding it in the future? Would you mind if I added it? And in that case, should there be a switch to hide it so Helsinki can keep using it as they do now?

**Comment by ahukkanen** [comment]
Yes, you are correct - it's by design. It was initially in the module but was later removed due to a change request.

The reasoning being: many people in the Finnish municipality sector (not only in Helsinki, all around) have a strong opinion that some people are left out from participation when their name is visible publicly. They'd rather just submit the idea on paper or tell it to the civil servants directly than publicly post it with their name attached to it. This is direct feedback from the citizens, some people have raised this concern. This one barrier for participation is removed by hiding their name - they feel more encouraged to participate.

If such feature is added, our request would be:
- Make it a configuration flag, preferably module level code configuration (disabled or enabled altogether for the whole module, not component specific)
- By default, the configuration should be disabled.

As the module is specifically developed for Helsinki based on their needs, we cannot do any major design changes without their approval. These kinds of requests are a bit difficult in that sense that it is in conflict what has been requested by Helsinki.

**Comment by PierreMesure** [comment]
Thanks for all these details. I won't do it just now but your request is fully reasonable. And it can leave on as a fork or a branch until Helsinki accepts it.
